_posts/2017-01-30-inc-cmd-lessonslearned.md

```diff
@@ -0,0 +1,79 @@
+
+***
+
+#### DON'T PANIC!
+It’s just a computer system and a technical problem that needs solving, the only difference between this and a JIRA story is the manner of approach and doing it with urgency. Take a ‘Condor moment’ then crack on with systematically triaging your way through the issues and bringing the impacted services back online.
+
+#### The first responder is the first incident commander
+The first person on the scene is by default in control of the incident, until until they get more help or can safely hand it off to someone else. In some cases this may mean they are best focusing on raising the alarm rather than immediately fighting the fire, but in all cases they need to focus effort to gain control of the situation.
+
+#### Seize the initiative
+Make the mental switch from ‘peace’ to ‘war’ and do not allow the situation and events that are occurring to dictate the pace and direction of the incident. A calm and confident incident commander takes control of the situation and issues clear and explicit directives to his or her available technical resource in order to triage the incident and drive it towards resolution as quickly and safely as possible with the minimum amount of collateral damage. This is often the difference between a ten minute incident and a two hour incident.
+
+#### IC’s do not fix problems
+As coordinators of resource the IC should not be at the terminal issuing commands. It is the role of the IC to set the strategic tone for the incident and leave the individual tactics to the subject matter experts at their disposal. In many cases, the IC has the team debate the pro's and con's of a particular solution – but the IC makes the final call based on the best available information to them from the SME's at the time.
+
+#### Have a single point of contact for each tribe/function
+Best case scenario, the single point of contact in a team will be that teams subject matter expert who has also trained as an IC, they will understand the same processes the IC is going through and have information on hand before the IC needs it. Often this is the difference between the IC being able to maintain control of an incident or it descending into floods of Slack messages from 10 people in each team which will rapidly overwhelm them, and inevitably slows down the resolution of the incident. Often the fastest way to get the relevant single point of contact for each team is request that they dial into a conference call, people tend to be more reluctant to shout over the IC or others on a call
+
+#### Keep a log
+Whether on paper or on a computer. Developing and rehearsing your distributed ways of working and communicating using a tool like Slack will help with this if you keep the channel updated on what is going on. If you have taken your incident response to a conference call summarise it for the people not on the call regularly.
```

Comment:
Dangling modifier! Do you summarise regularly or are people not on the call regularly? Consider rephrasing.